caluma/caluma_form/schema.py

```diff
@@ -751,6 +766,10 @@ class Meta:
 
 class ListAnswer(AnswerQuerysetMixin, FormDjangoObjectType):
     value = graphene.List(graphene.String)
+    selected_options = ConnectionField(SelectedOptionConnection)
+
+    def resolve_selected_options(self, info, **args):
```

Comment:
Not entirely sure, it's been a while since I messed with the schema, but:  Is the resolve method really needed here? It only returns `self.selected_options` anyway.

Reply:
Right, done.